studiopress/pattern-manager
# Pass values directly instead of postMeta object for editor component props

This PR aims to perform some cleanup on the way props are passed to components in meta controls. Also, some nitpicky cleanup is done to alphabetize and otherwise organize lists of type properties, component props, etc.

To borrow from the comment in #88:

> I think we should consider rethinking the passing of `postMeta` to every component, instead getting the values we need from `useEditedPostData` and passing them directly as props.
> 
> For example, instead of this:
> 
> ```js
> <KeywordsPanel
>     postMeta={ postMeta }
>     handleChange={ updatePostMeta }
> />
> <DescriptionPanel
>     postMeta={ postMeta }
>     handleChange={ updatePostMeta }
> />
> ```
> 
> ...we could do something like this:
> ```js
> <KeywordsPanel
>     keywords={ metaKeywords }
>     handleChange={ updatePostMeta }
> />
> <DescriptionPanel
>     description={ metaDescription }
>     handleChange={ updatePostMeta }
> />
> ```

---

### How to test
Not really needed, but if you want to test:
1. Checkout the branch
2. Make sure things fundamentally work the same as before in the editor
